roles/kubernetes-apps/argocd/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 - name: Kubernetes Apps | Install yq
   become: yes
   get_url:
-    url: "https://github.com/mikefarah/yq/releases/download/v4.25.3/yq_linux_amd64"
+    url: "https://github.com/mikefarah/yq/releases/download/v4.26.1/yq_linux_amd64"
```

Comment:
Is this change really needed for the new argo version or did this change get picked up in this PR by mistake?

Reply:
Not needed but keeping 'yq' up to date.

Reply:
This should at least be in a separate commit or the change indicated in the commit message.

Reply:
It does not block the purpose of the update If not updated. So I will remove it.